source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -94,9 +93,9 @@ void renderBackgroundImage(const std::shared_ptr<AdaptiveCards::BackgroundImage>
             imgView.translatesAutoresizingMaskIntoConstraints = NO;
             [containerView insertSubview:imgView atIndex:0];
 
-            if (img) {
+            if (imgView.image) {
                 // apply now if image is ready, otherwise wait until it is loaded
-                applyBackgroundImageConstraints(backgroundImage.get(), imgView, img);
+                applyBackgroundImageConstraints(backgroundImage.get(), imgView, imgView.image);
```

Comment:
question here -- is `applyBackgroundImageConstraints` always passed a `UIImageView` that has a valid `.image`? should `applyBackgroundImageConstraints` just use the image property rather than having it passed in as a parameter?

Reply:
your question is a valid one. There is a case SDK can have a UIImageview and UIImage separately, but such cases are rare. I will look into this when I refactor the background image rendering which is due soon.

Reply:
sounds good 👍